src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/TagHelperDescriptorExtensions.cs

```diff
@@ -152,6 +152,18 @@ public static bool IsRefTagHelper(this TagHelperDescriptor tagHelper)
                 string.Equals(ComponentMetadata.Ref.TagHelperKind, kind);
         }
 
+        public static bool IsComponentFullyQualifiedNameMatch(this TagHelperDescriptor tagHelper)
```

Comment:
I've been adding doc comments to these lately just because they are extension methods and they show up all over. Not going to player hate if you don't want to tho.

Reply:
Can do.